dashboard/src/RemoteComponent.js

```diff
@@ -0,0 +1,6 @@
+import { createRemoteComponent, createRequires } from "@paciolan/remote-component";
+import { resolve } from "./remote-component.config.js";
+
+const requires = createRequires(resolve);
+
+export const CustomComponent = createRemoteComponent({ requires });
```

Comment:
can you use Typescript here instead? (not sure if it's not possible for some reason)

Reply:
I spent sometime looking into this and the package doesnt seem to support a typescript implementation yet (they plan on adding it according to their docs). I can add a TODO there for when the package is updated (they seems to update it quite frequently)